src/interactive/repl.ts

```diff
@@ -272,7 +251,7 @@ async function selectJuliaBlock() {
     }
 }
 
-async function executeJuliaCellInRepl() {
+async function executeJuliaCellInRepl(shouldMove: boolean = false) {
```

Comment:
so currently `shouldMove` parameter doesn't seem to be respected anyway, right ?
I think currently this function will always behave as `run-cell-and-move`.

Reply:
Good point. Fixed.

Reply:
confirmed ;)